src/Paramore.Brighter.Extensions.DependencyInjection/ServiceCollectionBrighterBuilder.cs

```diff
@@ -185,11 +185,11 @@ public IBrighterBuilder TransformsFromAssemblies(params Assembly[] assemblies)
                 throw new ArgumentException("Value cannot be an empty collection.", nameof(assemblies));
 
             var transforms =
-                from ti in assemblies.SelectMany(a => a.DefinedTypes).Distinct()
+                from ti in assemblies.SelectMany(GetLoadableTypes).Distinct()
                 where ti.IsClass && !ti.IsAbstract && !ti.IsInterface
-                from i in ti.ImplementedInterfaces
+                from i in ti.GetInterfaces()
```

Comment:
The change here is due to using `Type` over `TypeInfo`. `TypeInfo` is no longer something we need to use